examples/analytics/emr-on-eks/main.tf

```diff
@@ -1,62 +1,40 @@
-terraform {
-  required_version = ">= 1.0.1"
-
-  required_providers {
-    aws = {
-      source  = "hashicorp/aws"
-      version = ">= 3.66.0"
-    }
-    kubernetes = {
-      source  = "hashicorp/kubernetes"
-      version = ">= 2.6.1"
-    }
-    helm = {
-      source  = "hashicorp/helm"
-      version = ">= 2.4.1"
-    }
-  }
-
-  backend "local" {
-    path = "local_tf_state/terraform-main.tfstate"
-  }
+provider "aws" {
+  region = local.region
 }
 
-provider "aws" {}
-
 provider "kubernetes" {
-  experiments {
-    manifest_resource = true
```

Comment:
we don't need this anymore?

Reply:
I don't believe so, no - this has been available since 2.4 and it looked like the lowest version we had set was 2.7

https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs/guides/alpha-manifest-migration-guide

Also, I don't think we even have any `kubernetes_manifest` resources in use in the repo